### DISCUSSION TITLE
Will create-nuxt-app support Nuxt 3?

### DISCUSSION THREAD

**Discussion by zhaoweizhong** [nuxt/create-nuxt-app#927] [Q&A]
Will `create-nuxt-app` support Nuxt 3? If so, is there a timeline?

**Comment by katerlouis** [comment]
Also wondering; 

In this discussion it's not clear what the final decision on this is:
https://github.com/nuxt/framework/discussions/3986

**Comment by katerlouis** [comment]
Any news here? Nuxt 3 is releasing in less than a week; to me and many others the wizard from `create-nuxt-app` is a vital part.

**Comment by katerlouis** [comment]
I'm surprised that no one "in charge" is reacting to this very natural question? It's odd that such a powerful feature is just hanging there and no one talks about it. Even @Atinux agrees that we need an init-wizard like create-nuxt-app back: https://github.com/nuxt/starter/issues/134

It doesn't reflect well on the Nuxt ecosystem and community when questions like this are being ignored for almost a year.

So again: Is it really necessary to start from scratch or can create-nuxt-app be upgraded (or forked under a new name) to support Nuxt3 aswell? What are the future plans for `create-nuxt-app`?
